src/export/ValueSetExporter.ts

```diff
@@ -32,8 +32,8 @@ export class ValueSetExporter {
     if (fshDefinition.description) {
       valueSet.description = fshDefinition.description;
     }
-    // Version is set to value provided in config, will be overriden if reset by rules
-    valueSet.version = this.tank.config.version;
+    delete valueSet.version; // deleting to allow the IG Publisher default to take hold
+    valueSet.status = this.pkg.config.status;
```

Comment:
As noted in a previous comment: Unless there is a reason to use `this.pkg.config`, we should use `this.tank.config` since that's where we pull the config from when setting other config-based values in this same function.

Reply:
Addressed In commit [25e2afa](https://github.com/FHIR/sushi/pull/1143/commits/25e2afad1a873bf611c84ecca118ffc500f0d74e).